### PR TITLE
[BottomNavigation] Add separate property for selected title color

### PR DESCRIPTION
**Pull request by jmdetloff**
Exposes a selected item title color. Does not implement the full title color image color for state pattern as MDCNavigationBar does not display a title in its unselected state, which made just this single property addition seem cleaner. 

https://www.pivotaltracker.com/story/show/156742634

### CODE REVIEW

File: components/BottomNavigation/src/MDCBottomNavigationBar.m [hunk 1]
```diff
@@ -502,6 +504,7 @@ - (void)setTitleBelowItem:(BOOL)titleBelowItem {
 
 - (void)setSelectedItemTintColor:(UIColor *)selectedItemTintColor {
   _selectedItemTintColor = selectedItemTintColor;
+  _selectedItemTitleColor = selectedItemTintColor;
```
Comment (jverkoey) [inline components/BottomNavigation/src/MDCBottomNavigationBar.m:507]: Should this call setSelectedItemTitleColor?

**Review by jverkoey — CHANGES_REQUESTED** [review]

**Review by jverkoey — DISMISSED** [review]
Please add a test that verifies that selectedItemTitleColor propagates to the relevant views.

File: components/BottomNavigation/src/MDCBottomNavigationBar.m (hunk 1 above)
Comment (jmdetloff) [reply components/BottomNavigation/src/MDCBottomNavigationBar.m:507]: Nope, this should take care of everything. Setting itemView.selectedItemTintColor updates the itemView's text label color

**Review by jverkoey — APPROVED** [review]

File: components/BottomNavigation/tests/unit/BottomNavigationBarColorThemerTests.m [hunk 2]
```diff
@@ -18,6 +18,7 @@
 #import "MDCBottomNavigationBarColorThemer.h"
 #import "MaterialColorScheme.h"
 #import "MaterialBottomNavigation.h"
+#import "../../src/private/MDCBottomNavigationItemView.h"
```
Comment (jverkoey) [inline components/BottomNavigation/tests/unit/BottomNavigationBarColorThemerTests.m:21]: I'd be surprised if this builds with bazel. You may have to add a "private" target for your unit tests.

Example:

```python
mdc_objc_library(
    name = "private",
    hdrs = native.glob(["src/private/*.h"]),
    includes = ["src/private"],
    visibility = ["//visibility:private"],
)

swift_library(
    name = "unit_test_swift_sources",
    srcs = glob(["tests/unit/*.swift"]),
    deps = [
        ":Snackbar",
        ":private",
        ":ColorThemer",
        ":TypographyThemer",
    ],
    visibility = ["//visibility:private"],
)
```

Comment (jmdetloff) [reply components/BottomNavigation/tests/unit/BottomNavigationBarColorThemerTests.m:21]: There already is a private target our tests depend on, we're already importing this header within BottomNavigationTests.m

**Pull request merged**